cgi/product_jqm_multilingual.pl

```diff
@@ -373,6 +373,22 @@
 
 			my $modifier = undef;
 
+			# energy: (see bug https://github.com/openfoodfacts/openfoodfacts-server/issues/2396 )
+			# 1. if energy-kcal or energy-kj is set, delete existing energy data
+			if (($nid eq "energy-kj") or ($nid eq "energy-kcal")) {
```

Comment:
- vs _ ?

Reply:
All nutrient identifiers use a dash. e.g. saturated-fat. Then we add _things with underscore like _unit, _value, _100g, _serving etc.